resources/lib/player_listener.py

```diff
@@ -38,7 +38,7 @@ def _sanity_check_segments(segments):  # type: (Iterable[SponsorSegment]) -> boo
 
 def get_sponsor_segments(
     api, video_id
-):  # type: (SponsorBlockAPI, str) -> Optional[List[SponsorSegment]]
+):  # type: (SponsorBlockAPI, str) -> Optional[list[SponsorSegment]]
```

Comment:
FYI: the reason all the types were capitalized is because support for standard library type generics was only added in 3.9.